packages/expo-yarn-workspaces/bin/check-workspace-dependencies.js

```diff
@@ -91,5 +95,8 @@ function countMismatchedWorkspaceDependencies(workspacesInfo) {
 }
 
 if (module === require.main) {
-  checkWorkspaceDependencies();
+  let passed = checkWorkspaceDependencies();
+  if (!passed) {
```

Comment:
Maybe this way?
```suggestion
  if (!checkWorkspaceDependencies()) {
```

Also, if you don't modify a reference then I'd prefer to always use `const` over `let` so it's clear the reference is not gonna change

Reply:
This seems fine here but in other cases want to treat `await` more as a statement than an expression. `let x = await ...; if (x)` instead of `if (await ...)`